public/index.pug

```diff
@@ -29,17 +29,33 @@ html(lang="en-US")
         li(class="header__list-item")
-          a(href="https://unpkg.com/browse/simple-icons/") Unpkg
+          a(class="header__list-icon" title="Unpkg (Content Delivery Network)" href="https://unpkg.com/browse/simple-icons/")
+            svg(role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg")
+              path(d="M12-.0001a12 12 0 0 0-12 12 12 12 0 0 0 12 12 12 12 0 0 0 12-12 12 12 0 0 0-12-12zm0 2.00178A10.00892 10.00892 0 0 1 22.00893 12.0106 10.00892 10.00892 0 0 1 12 22.01953 10.00892 10.00892 0 0 1 1.99108 12.0106 10.00892 10.00892 0 0 1 12 2.00168zm2.14095 3.92864-.03126 4.04638-.0333 4.0464c-.73005 2.10882-3.4148 2.10882-3.99286.007-.07493-.31042-.11775-1.79838-.11775-4.28188V5.93734H6.84l.04314 4.14273c.05342 4.47458.08446 4.7422.66155 5.90901.35326.6851 1.20964 1.44514 1.96967 1.72346 1.39162.52453 3.68243.52453 5.07405 0 .74933-.27832 1.55218-.9752 1.93755-1.68064.58876-1.10259.65406-1.62712.65406-6.01606l.0098-4.01427-3.04014-.06466z")
         li(class="header__list-item")
-          a(href="https://github.com/simple-icons/simple-icons/blob/develop/DISCLAIMER.md") Disclaimer
+          a(class="header__list-icon" title="Open Collective" href="https://opencollective.com/simple-icons")
+            svg(role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg")
+              path(d="M12 0C5.373 0 0 5.373 0 12s5.373 12 12 12c2.54 0 4.894-.79 6.834-2.135l-3.107-3.109a7.715 7.715 0 1 1 0-13.512l3.107-3.109A11.943 11.943 0 0 0 12 0zm9.865 5.166l-3.109 3.107A7.67 7.67 0 0 1 19.715 12a7.682 7.682 0 0 1-.959 3.727l3.109 3.107A11.943 11.943 0 0 0 24 12c0-2.54-.79-4.894-2.135-6.834z")
+        li(class="header__list-item")
+          a(class="header__list-icon" title="Legal disclaimer" href="https://github.com/simple-icons/simple-icons/blob/master/README.md")
```

Comment:
Shouldn't this link to the `DISCLAIMER.md` file?

Reply:
huh. yes, yes it should. I didn't remove the correct link when removing the duplicate Readme, apparently.

Reply:
Feel free to merge once you fixed it.